connector/src/main/java/org/geysermc/connector/network/translators/item/ItemTranslator.java

```diff
@@ -92,18 +95,19 @@ public static void init() {
         NBT_TRANSLATORS = loadedNbtItemTranslators.keySet().stream().sorted(Comparator.comparingInt(loadedNbtItemTranslators::get)).collect(Collectors.toList());
     }
 
-    public static ItemStack translateToJava(ItemData data) {
+    public static ItemStack translateToJava(ItemData data, int protocolVersion) {
```

Comment:
I would just add GeyserSession as the first variable here, in the event that future changes require it.

Reply:
We can't provide a GeyserSession in the recipe registry loader.

Reply:
Riiiiiiight